functional-tests/support/pagePaths.ts

```diff
@@ -1,8 +1,8 @@
 export const pagePaths = {
 	"published guidance list": "/guidance/published",
 	"in consultation guidance list": "/guidance/inconsultation",
-	"in developmen guidancet list": "/guidance/indevelopment",
-	"proposed guidance list": "/guidance/proposed",
+	"in development guidance list": "/guidance/indevelopment",
```

Comment:
change route from "indevelopment"  to "in-development", matching "awaiting-development"

Reply:
Yeah that'd be nice, but it means adding extra routes in varnish and redirects...which I'm not going to do as part of this card. But it would certainly be nice! We use hyphen separated routes elsewhere so the indevelopment and inconsultaiton ones do look odd now